### PR TITLE
Update website/docs/r/logpush_job.html.markdown

### PR DESCRIPTION
**Pull request by fclerg**
- Fixing confusing name field in the example as it must not contain spaces
- Adding missing required field and their description.
- Removing import section as it is not supported for this resource. Error message when trying to run an import : 
```
cloudflare_logpush_job.example_job: Importing from ID "a9c1ccd9b3d737cc0a83e54226b6e96d/6080"...

Error: cloudflare_logpush_job.example_job (import id: a9c1ccd9b3d537cc0a63e54226b6e96d/6070): import cloudflare_logpush_job.example_job (id: a9c1ccd9b3d537cc0a63e54226b6e96d/6080): resource cloudflare_logpush_job doesn't support import
```

### CODE REVIEW

**Comment by fclerg** [comment]
applied your suggested changes @jacobbednarz, thanks. Not sure why had to open a new PR, sorry about that.

File: website/docs/r/logpush_job.html.markdown [hunk 1]
```diff
@@ -26,18 +27,13 @@ resource "cloudflare_logpush_job" "example_job" {
...
-* `ownership_challenge` - (Required) Ownership challenge token to prove destination ownership. See [https://developers.cloudflare.com/logs/tutorials/tutorial-logpush-curl/](https://developers.cloudflare.com/logs/tutorials/tutorial-logpush-curl/)
 
-## Import
+* `name` - (Required) The name of the logpush job to create. Must match the regular expression `^[a-zA-Z0-9\-\.]*$`
+* `zone_id` - (Required) The zone ID where the logpush job should be created.
+* `destination_conf` - (Required) Uniquely identifies a resource (such as an s3 bucket) where data will be pushed. Additional configuration parameters supported by the destination may be included. See [Logpush destination documentation](https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#destination)
+* `ownership_challenge` - (Required) Ownership challenge token to prove destination ownership. See [https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#usage](https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#usage)
```
Comment (patryk) [inline website/docs/r/logpush_job.html.markdown]:
```suggestion
* `ownership_challenge` - (Required) Ownership challenge token to prove destination ownership. See [Developer documentation](https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#usage).
```

**Comment by jimbobjim123** [comment]
Looking at the commit we are also removing the import logpush job option from the docs. Is this not supported anymore? Has it ever been an option?

**Comment by ygt-francoisc** [comment]
As mentioned above here is the message I get when trying to import a logpush resource :
```
cloudflare_logpush_job.example_job: Importing from ID "a9c1ccd9b3d737cc0a83e54226b6e96d/6080"...

Error: cloudflare_logpush_job.example_job (import id: a9c1ccd9b3d537cc0a63e54226b6e96d/6070): import cloudflare_logpush_job.example_job (id: a9c1ccd9b3d537cc0a63e54226b6e96d/6080): resource cloudflare_logpush_job doesn't support import
```

Not sure if it has ever been supported. I guess not as the _cloudflare_logpush_job_ resource has been introduced quite recently (see issue #210)

**Comment by fclerg** [comment]
As mentioned above here is the message I get when trying to import a logpush resource :
```
cloudflare_logpush_job.example_job: Importing from ID "a9c1ccd9b3d737cc0a83e54226b6e96d/6080"...

Error: cloudflare_logpush_job.example_job (import id: a9c1ccd9b3d537cc0a63e54226b6e96d/6070): import cloudflare_logpush_job.example_job (id: a9c1ccd9b3d537cc0a63e54226b6e96d/6080): resource cloudflare_logpush_job doesn't support import
```

Not sure if it has ever been supported. I guess not as the _cloudflare_logpush_job_ resource has been introduced quite recently (see issue #210)

**Comment by jacobbednarz** [comment]
Logpush doesn’t have support for importing at the moment.

**Comment by jacobbednarz** [comment]
Thank you @fclerg!

**Pull request merged**